test/w_http_client_integration_test.dart

```diff
@@ -106,5 +106,27 @@ void main() {
       Map data = JSON.decode(await response.asText());
       expect(data['headers']['x-configured'], equals('true'));
     });
+
+    group('should set the withCredentials flag', () {
```

Comment:
I'm not sure I understand this test.  
I expected to see an assert statement or expects statement.
Can you explain a little around what this is testing and how it works?